### ISSUE TITLE
Add regex and/or glob support

### ISSUE THREAD

**Issue by echenley** [zalmoxisus/redux-devtools-filter-actions#5]
I'd imagine quite a few people are using this monitor to manage actions generated by Redux Form or other libraries. With Redux Form, the actions all have the signature `redux-form/ACTION`, so I end up having to do:

``` js
<FilterMonitor blacklist={ [
  'redux-form/CHANGE',
  'redux-form/BLUR',
  'redux-form/FOCUS',
  'redux-form/DESTROY',
  'redux-form/STOP_SUBMIT',
  // other actions
] }>
```

It would be nice to be able to do something like:

``` js
<FilterMonitor blacklist={ [
  /redux-form\//,
  // other actions
] }>
```

**Comment by zalmoxisus** [comment]
Yes, we have the same feature request [here](https://github.com/zalmoxisus/redux-devtools-extension/issues/45). Will implement it.

**Comment by zalmoxisus** [comment]
Implemented in `v1.1.0`, but regex should be passed as a string in order not to be breaking change. 
So `redux-form/` will dismiss all the actions which contains this string.
Thanks for the suggestion!

**Comment by echenley** [comment]
Yes! Thank you! :+1:
